### PR TITLE
fix(lean-info): `lean-show-goal` no longer interrupts `lean-find-defi…

### PR DESCRIPTION
**Pull request by cipher1024**
…nition`

### CODE REVIEW

File: lean-info.el [hunk 1]
```diff
@@ -101,6 +101,7 @@
 (defun lean-find-definition ()
   "Jump to definition of thing at point"
   (interactive)
+  (setq lean-show-goal--handler-mask t)
```
Comment (Kha) [inline lean-info.el]: Could you add a short comment here on why we need to mask the next invocation?

File: lean-type.el [hunk 2]
```diff
@@ -86,16 +86,20 @@
...
-      (lean-get-info-record-at-point
-       (lambda (info-record)
-         (let ((state (plist-get info-record :state)))
-           (unless (or (s-blank? state) (s-blank? (s-trim state)))
-             (lean-with-info-output-to-buffer lean-show-goal-buffer-name (princ state)))))))))
+  (if lean-show-goal--handler-mask
+      (setq lean-show-goal--handler-mask nil)
+    (let ((deactivate-mark)) ; keep transient mark
+      (when (and (not (eq lean-server-check-mode 'nothing))
+					; TODO(gabriel): revisit ^^ once info no longer reparses the file
```
Comment (Kha) [inline lean-type.el:96]: formatting

Comment (cipher1024) [reply lean-type.el:96]: This is the way emacs indents the code by default. How should I reformat it?

**Comment by Kha** [comment]
Okay, then I guess it's fine.

**Comment by cipher1024** [comment]
I think it should be good now

**Pull request merged**